theme_paptic/views/customizations.xml

```diff
@@ -206,11 +206,11 @@
 <!-- ==== Three Columns ===== -->
 <template id="s_three_columns" inherit_id="website.s_three_columns" name="Paptic s_three_columns">
     <xpath expr="//section" position="attributes">
-        <attribute name="class" remove="bg-200 pt32 pb32 o_cc o_cc3" add="pt120 pb120" separator=" "/>
-        <attribute name="data-oe-shape-data">{"shape":"web_editor/Origins/01","flip":[]}</attribute>
+        <attribute name="class" remove="bg-200 pt32 pb32" add="pt120 pb120 o_cc1 bg-o-color-2" separator=" "/>
```

Comment:
This will be in conflict with another task we have already fixing/chaning that. But for the record, what you do here is not ok: you cannot have a o_ccX class and a bg-o-color-X class at the same time, so remove this change :wink:

Reply:
@qsm-odoo 
Ok I removed the change. But without that, the shape is hidden (it's white on a white background).
Should I let like that ?